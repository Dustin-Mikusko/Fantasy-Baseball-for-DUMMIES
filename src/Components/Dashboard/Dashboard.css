.dashboard {
  margin: auto;
  text-align: center;
  display: flex;
  flex-direction: column;
}

.dashboard-welcome {
  margin-top: -5px;
  font-size: 3em;
  margin-top: 150px;
}

.dashboard-info {
  font-size: 1.5em;
  width: 50%;
  margin: auto;
  margin-bottom: 40px;
}

.dashboard-link {
  border-radius: 3px;
  padding-top: 12px;
  text-decoration: none;
  margin: auto;
  margin-bottom: 15px;
  width: 50%;
  height: 60px;
  font-size: 2.3em;
  color: white;
  background-color: #041f42
}

.dashboard-link:hover {
  background-color: rgb(6, 71, 155);
}

.dashboard-link:active {
  transform: scale(.97)
}
