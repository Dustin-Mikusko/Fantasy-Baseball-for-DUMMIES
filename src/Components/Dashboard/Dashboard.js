import React from 'react';
import './Dashboard.css';
import { Link } from 'react-router-dom';
import Header from '../Header/Header';

export const Dashboard = () => {
  return (
    <>
      <section className='dashboard'>
        <Header />
        <h1 className='dashboard-welcome'>Welcome to your dashboard!</h1>
        <h4 className='dashboard-info'>From here you can search for players by name, see the top 25 players by each stat, or view the players that you have favorited!</h4>
        <Link to='search' className='dashboard-link'>SEARCH PLAYERS</Link>
        <Link to='top-players' className='dashboard-link'>VIEW TOP 25</Link>
        <Link to='favorites' className='dashboard-link'>VIEW YOUR PLAYERS</Link>
      </section>
    </>
  )
}

export default Dashboard;
