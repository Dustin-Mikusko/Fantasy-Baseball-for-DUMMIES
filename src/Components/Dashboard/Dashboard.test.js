import React from 'react';
import Dashboard from './Dashboard';
import { shallow } from 'enzyme';


describe('Dashboard', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Dashboard />);

    expect(wrapper).toMatchSnapshot();
  })
})
