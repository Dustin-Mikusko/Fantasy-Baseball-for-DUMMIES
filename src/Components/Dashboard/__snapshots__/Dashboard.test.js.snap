// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard should match the snapshot 1`] = `ShallowWrapper {}`;
